nautobot/extras/tests/test_api.py

```diff
@@ -2526,3 +2526,43 @@ def test_create_webhooks_with_same_content_type_same_url_common_action(self):
             response.data[0]["type_create"][0],
             "A webhook already exists for create on dcim | device type to URL http://example.com/test1",
         )
+
+    def test_patch_webhooks_with_same_content_type_same_url_common_action(self):
+        self.add_permissions("extras.change_webhook")
+
+        self.webhooks[2].payload_url = self.webhooks[1].payload_url
+        self.webhooks[2].save()
+
+        data = {"type_update": True}
+
+        response = self.client.patch(self._get_detail_url(self.webhooks[2]), data, format="json", **self.header)
+        self.assertHttpStatus(response, status.HTTP_400_BAD_REQUEST)
+        self.assertEquals(
+            response.data["type_update"][0],
+            f"A webhook already exists for update on dcim | device type to URL {self.webhooks[1].payload_url}",
+        )
```

Comment:
Additional negative tests to consider:

- changing the payload_url on an existing webhook so that it now conflicts with another webhook.
- changing the content_types on an existing webhook so that it now conflicts with another webhook.